build/source/driver/multi_driver.f90

```diff
@@ -211,23 +211,44 @@ program multi_driver
 integer(i4b)                     :: err=0                      ! error code
 character(len=1024)              :: message=''                 ! error message
 
+! subsetting the model run
+integer(i4b)                     :: strtHRU                    ! index of the starting hru for a subset run
+character(len=10)                :: strTMP                     ! local temp string
 ! *****************************************************************************
 ! (1) inital priming -- get command line arguments, identify files, etc.
 ! *****************************************************************************
-print*, 'start'
 ! get the initial time
-call date_and_time(cdate1,ctime1)
-print*,ctime1
+call date_and_time(values=ctime1)
+print "(A,I2,':',I2,':',I2)", 'start at ',ctime1(5:7)
+! check numbers of command-line arguments
+select case (iargc())
+ case (2)
+  !do nothing; this is a full run
+  strtHRU=0
```

Comment:
I think that it would be clearer to set BOTH `startHRU` and `nHRU` to the global missing integer, e.g.,

```
startHRU=integerMissing
nHRU=integerMissing
```

This will be important later